aws.sh

```diff
@@ -8,10 +8,18 @@ set -o pipefail
 trap - INT TERM
 
 docker run --rm \
-	-t $(tty &>/dev/null && echo "-i") \
-	-e "AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}" \
-	-e "AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}" \
-	-e "AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}" \
+	-t "$(tty >/dev/null 2>&1 && printf -- '-i')" \
+	-v "${AWS_SHARED_CREDENTIALS_FILE:-$HOME/.aws}:${AWS_SHARED_CREDENTIALS_FILE:-/root/.aws}" \
```

Comment:
Mount `~/.aws` in `/root/.aws`, otherwise if `AWS_SHARED_CREDENTIALS_FILE` is defined, mount it as is in the container